TDD_Training/TDD_Training/FizzBuzzService.cs

```diff
@@ -6,27 +6,51 @@ public class FizzBuzzService
     {
         public string CheckFizzBuzz(int givenNumber)
         {
-            switch(givenNumber)
+            try
             {
-                case int _ when IsMultipleOfThreeAndFive(givenNumber) == true:
+                if (givenNumber <= 0)
```

Comment:
You can invert if here. 
Also please send the necessary parameters in the Exception(message and number)

Reply:
Resolved both comments